server/storage/kv/kv.go

```diff
@@ -14,10 +14,16 @@
 
 package kv
 
+// RevisionUnavailable is the value of unavailable revision,
+// when the kv does not exist (etcd_kv), or is not supported (mem_kv & leveldb_kv).
+const RevisionUnavailable = -1
+
 // Base is an abstract interface for load/save pd cluster data.
 type Base interface {
 	Load(key string) (string, error)
 	LoadRange(key, endKey string, limit int) (keys []string, values []string, err error)
+	LoadRevision(key string) (string, int64, error)
```

Comment:
`revision` here is a very specific term that is related to etcd, is there a better way to avoid introducing it in this generic KV interface?

Reply:
I tried a little bit but I don't think there's a way to avoid modifying `Base` interface unless we change `etcdKVBase` to exported type.
I'm just not sure if thats the way to go, feels kinda hacky. What do you think? @JmPotato